Error at respond
Traceback (most recent call last):
  File "/home/site/wwwroot/EventGridTrigger/__init__.py", line 48, in main
    respond(event)
  File "/home/site/wwwroot/EventGridTrigger/__init__.py", line 112, in respond
    reply_message = think( event_json['message'], to_phone_number)
  File "/home/site/wwwroot/EventGridTrigger/__init__.py", line 72, in think
    message_log = remember(user)
  File "/home/site/wwwroot/EventGridTrigger/__init__.py", line 56, in remember
    from_messages = receive_from_servicebus(user)
  File "/home/site/wwwroot/EventGridTrigger/__init__.py", line 34, in receive_from_servicebus
    my_body = my_body.decode('utf-8')
AttributeError: 'str' object has no attribute 'decode'